logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -39,7 +44,10 @@ def update(time, snapshot)
       #   - We can use a synchronization mechanism between the called thread (update method)
       #   and the plugin thread (run method)
       #   - How we handle back pressure here?
-      @queue << snapshot.to_event
+      #   - one snashot should be only one event
```

Comment:
Yes, like discussed yesterday, I will do that in another PR, if its okay with you?